helix-view/src/clipboard.rs

```diff
@@ -17,7 +17,7 @@ pub trait ClipboardProvider: std::fmt::Debug {
 #[cfg(not(windows))]
 macro_rules! command_provider {
     (paste => $get_prg:literal $( , $get_arg:literal )* ; copy => $set_prg:literal $( , $set_arg:literal )* ; ) => {{
-        log::info!(
+        log::debug!(
```

Comment:
Looks like an unrelated change here

Reply:
Yeah, I made this debug because it makes the integration tests really noisy. This line gets printed for every single test.

Even outside of that context though, it seems like info that is really only important to print if you're trying to debug a problem with the clipboard.

What do you think though?

Reply:
Hmm yeah I think those could be debug: we have a `log::warn!` when there isn't a clipboard provider and this info shows up in `hx --health clipboard`. The current `log::info!` does seem a little noisy